lib/routes/charge/pos_invoice.dart

```diff
@@ -843,21 +859,19 @@ class POSInvoiceState extends State<POSInvoice> with TickerProviderStateMixin {
     });
   }
 
-  changeCurrency(
-      Sale currentSale, String value, UserProfileBloc userProfileBloc) {
+  changeCurrency(Sale currentSale, String value,
+      UserProfileBloc userProfileBloc, AccountModel accountModel) {
     setState(() {
-      Currency currency = Currency.fromTickerSymbol(value);
-
-      bool flipFiat = _useFiat == (currency != null);
-      if (flipFiat) {
-        _useFiat = !_useFiat;
-      }
+      CurrencyWrapper currency =
+          CurrencyWrapper.fromShortName(value, accountModel);
       _clearAmounts(currentSale);
 
-      if (currency != null) {
-        userProfileBloc.currencySink.add(currency);
+      if (currency.btc != null) {
```

Comment:
btc property is exposed by mistake, we shouldn't expose it. Use "isFiat" instead.

Reply:
[Done](https://github.com/breez/breezmobile/pull/309/commits/e7b2dfd7680ea9b850ed03f26e753a42e3a37c28)